docs/_toc.yml

```diff
@@ -26,6 +26,8 @@ parts:
       file: tutorials/visualize-embeddings
     - title: Clay MAE reconstruction
       file: tutorials/reconstruction
+    - title: NAIP Inference and Similarity Search with Clay
+      file: tutorials/v1-inference-simsearch-naip-stacchip
 - caption: Finetune examples
```

Comment:
At L39, could you fix a typo "Finetunig" -> "Finetuning"?

Reply:
done!